cfgrammar/src/lib/span.rs

```diff
@@ -51,13 +51,15 @@ pub struct NewlineToLineColCache<'a> {
 
 impl<'a> NewlineToLineColCache<'a> {
     pub fn new(src: &'a str) -> Self {
-        let newlines: Vec<usize> = src
-            .char_indices()
-            .filter_map(|c| match c {
-                (offset, '\n') | (offset, '\r') => Some(offset),
-                _ => None,
-            })
-            .collect::<Vec<_>>();
+        let mut newlines: Vec<usize> = vec![0];
```

Comment:
Not really a good place to add this note, but it occurred to me I need to check the old `newlines` vec it seems likely we can now remove them if the only uses are `push`

Reply:
One was unused, and the other could be replaced by a count of newlines used in some whitespace sensitive parsing.